### PR TITLE
Feature/registration interval config

### PR DESCRIPTION
**Pull request by javibookline**
Allow the user to control the minimum/default outbound registration expire intervals via two new envvars.

Keep the current defaults of 30s and 3600s if the envvars are not defined.

### CODE REVIEW

File: lib/sip-trunk-register.js [hunk 1]
```diff
@@ -114,7 +117,7 @@ class Regbot {
           else if (res.has('Expires')) {
             expires = parseInt(res.get('Expires'));
           }
-          if (isNaN(expires) || expires < 30) expires = DEFAULT_EXPIRES;
+          if (isNaN(expires) || expires < MIN_EXPIRES) expires = Math.max(MIN_EXPIRES, DEFAULT_EXPIRES);
```
Comment (davehorton) [inline lib/sip-trunk-register.js:120]: what is the scenario where MIN_EXPIRES will possibly be greater than DEFAULT_EXPIRES?

Comment (javibookline) [reply lib/sip-trunk-register.js:120]: if we enable setting DEFAULT_EXPIRES via envvar, well, someone could pass a lower value

Comment (javibookline) [reply lib/sip-trunk-register.js:120]: it's dumb, but I figured if a minimum is set, then we should comply with it

Comment (davehorton) [reply lib/sip-trunk-register.js:120]: why would we allow a default expires value to be less than the minimum?  I don't believe in coding to support a case like that, we should rather reject attempts to set meaningless values such as these

Comment (javibookline) [reply lib/sip-trunk-register.js:120]: at what level should those attempts be rejected? do we kill the process when loading the envvars? or do we just specify it in the docs and let the code run?

Comment (javibookline) [reply lib/sip-trunk-register.js:120]: https://github.com/jambonz/helm-charts/pull/36 includes these new envvars in the helm charts too

Comment (javibookline) [reply lib/sip-trunk-register.js:120]: @davehorton would you rather leave it as it was?

`          if (isNaN(expires) || expires < MIN_EXPIRES) expires = DEFAULT_EXPIRES;`

Comment (davehorton) [reply lib/sip-trunk-register.js:120]: I would say if expires < MIN_EXPIRES log a message and set it to MIN_EXPIRES

Comment (javibookline) [reply lib/sip-trunk-register.js]: @davehorton is this more in line with what you had in mind?

https://github.com/jambonz/sbc-sip-sidecar/pull/50/commits/89832c8b0ea9085a6f09641ef515b1ec120e9175

**Pull request merged**